src/component/admin/billing/BillingGrid/BillingGrid.tsx

```diff
@@ -0,0 +1,325 @@
+
+    const freeAssigned = useMemo(
+        () => Math.min(users.length, seats),
+        [users, seats]
+    );
+
+    const paidAssigned = useMemo(
+        () => users.length - freeAssigned,
+        [users, freeAssigned]
+    );
+
+    const paidAssignedPrice = useMemo(
+        () => price.user * paidAssigned,
+        [price.user, paidAssigned]
+    );
+
+    const finalPrice = useMemo(
+        () => price.pro + paidAssignedPrice,
+        [price.pro, paidAssignedPrice]
+    );
+
+    const inactive = useMemo(
+        () => instanceStatus?.state !== InstanceState.ACTIVE,
+        [instanceStatus]
+    );
+
+    return (
+        <>
+            <Grid container spacing={4}>
+                <Grid item xs={12} md={5}>
+                    <StyledInfoBox sx={{ backgroundColor: colors.grey[200] }}>
+                        <Typography
+                            variant="body1"
+                            sx={{ marginBottom: '32px' }}
+                        >
+                            Billing Information
+                        </Typography>
+                        <ConditionallyRender
+                            condition={inactive}
+                            show={
+                                <Alert
+                                    severity="warning"
+                                    sx={{ marginBottom: '32px' }}
+                                >
+                                    In order to <strong>Upgrade trial</strong>{' '}
+                                    you need to provide us your billing
+                                    information.
+                                </Alert>
+                            }
+                        />
+                        <BillingInfoButton update={!inactive} />
+                        <StyledInfoLabel>
+                            {inactive
+                                ? 'Once we have received your billing information we will upgrade your trial within 1 business day'
+                                : 'These changes may take up to 1 business day and they will be visible on your next invoice'}
+                        </StyledInfoLabel>
+                        <Divider
+                            sx={{
+                                margin: '20px 0',
+                                borderColor: colors.grey[400],
+                            }}
+                        />
+                        <StyledInfoLabel>
+                            <a href="mailto:elise@getunleash.ai?subject=PRO plan clarifications">
+                                Get in touch with us
+                            </a>{' '}
+                            for any clarification
+                        </StyledInfoLabel>
+                    </StyledInfoBox>
+                </Grid>
+                <Grid item xs={12} md={7}>
+                    <StyledInfoBox
+                        sx={{
+                            boxShadow: '0px 1px 20px rgba(45, 42, 89, 0.1)',
```

Comment:
Might be worth theming this. Also, couldn't this go into the StyledInfoBox setup?

Reply:
Reasoning was: We have 2 StyledInfoBox that share common base styles. However each of them have specific style properties. So it made sense to me at the time to create a base `StyledInfoBox` and add the specific styles as `sx`.

In this case, would you prefer to see 2 separate `styled` components for each box, already including their specific properties?

Either way I added this boxShadow to theme 👍

Reply:
Should no longer be relevant with: https://github.com/Unleash/unleash-frontend/pull/993/commits/228f07b549a783d7cee4f8c80deb206983a1731b